source/features/copy-code-block.tsx

```diff
@@ -0,0 +1,38 @@
+import './copy-code-block.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import copyToClipboard from 'copy-text-to-clipboard';
+import features from '../libs/features';
+import {clippy} from '../libs/icons';
+
+function handleClick({currentTarget: button}: React.MouseEvent<HTMLButtonElement>): void {
+	if (button.previousSibling && button.previousSibling.textContent) {
+		copyToClipboard(button.previousSibling.textContent);
+	}
+}
+
+function renderButton(): void {
+	for (const codeBlock of select.all('code')) {
+		console.log(codeBlock);
+		codeBlock
+			.after(
+				<button className="copy-code-block btn-link" aria-label="Copy the code block" onClick={handleClick}>
+					{clippy()}
+				</button>
+			);
+	}
+}
+
+function init(): void {
+	if (select.exists('code')) {
+		renderButton();
```

Comment:
The whole `renderButton` function code can be placed inside the `init`.
And the extra `if (select.exists('code'))` check is redundant with the for-loop.